changelog.dd

```diff
@@ -71,6 +71,9 @@ $(BUGSTITLE Library Changes,
     $(LI $(RELATIVE_LINK2 emplace-inner-class, `std.conv.emplace` no longer allows
     to emplace classes directly nested inside other classes without specifying a
     suitable `outer` pointer))
+    $(LI Conversions from strings to number types using $(REF to, std,conv)
+        and $(REF parse, std,conv) were optimized. On average, integer types
+        are three times faster, and floating point types are 40% faster.)
```

Comment:
change log should be updated. ping @wilzbach . You wanted to change current change log system